server/app.ts

```diff
@@ -38,10 +38,10 @@ export default function createApp(userService: UserService): express.Application
   app.use(authorisationMiddleware())
 
   // App routes
-  app.use('/', homeRoutes(standardRouter(userService)))
   app.use('/select-location', selectLocationRoutes(standardRouter(userService)))
   app.use('/incentive-summary/:locationPrefix', incentivesTableRoutes(standardRouter(userService)))
   app.use('/prisoner-images/:imageId.jpeg', prisonerImagesRoutes(imageRouter()))
+  app.use('/', homeRoutes(standardRouter(userService)))
```

Comment:
Move this after the `/prisoner-images/:imageId.jpeg` route avoid the unnecessary calls.